src/main/java/validator/TestExceptionImpl.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2022-2022 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package validator;
+
+import io.fixprotocol.orchestra.message.TestException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TestExceptionImpl extends TestException {
```

Comment:
I think it is better to create our own exception and do not use the `TestException` from FIX orchestra at all

Reply:
could you please explain which advantages it gives us? we should copy the details from testException or we don't want to use them at all?

Reply:
Sorry, maybe you are right. We still need a field to store the details and the `TestException` already has them. So, let's keep it as is for now